onnx_tf/backend.py

```diff
@@ -453,6 +453,50 @@ def handle_constant(cls, node, input_dict):
     return [tf.constant(elements, dtype=dtype, shape=value.dims)]
 
   @classmethod
+  def _conv(cls, node, input_dict, transpose=False):
+    x = input_dict[node.inputs[0]]
+    x_rank = len(x.get_shape())
+    data_format = "NCDHW"
+    if x_rank == 3:
+      data_format = "NCW"
+    elif x_rank == 4:
+      data_format = "NCHW"
+    in_weights = input_dict[node.inputs[1]]
+    weights_rank = len(in_weights.get_shape())
+    if transpose:
+      # Translate weights from (M x C x KH x KW) to (KH x KW X C X M)
+      perm = list(range(2, weights_rank)) + [1, 0]
+    else:
+      # Translate weights from (C x M x KH x KW) to (KH x KW X C X M)
```

Comment:
Hi Arpith, I think you get this one the other way around. Normal case (non-transpose) is when we have (M x C x kH x kW).